docs/examples/couchbase.md

```diff
@@ -0,0 +1,271 @@
+<!--[metadata]>
++++
+title = "Dockerizing a Couchbase service"
+description = "Sharing data between 2 couchdb databases"
+keywords = ["docker, example, package installation, networking, couchbase"]
+[menu.main]
+parent = "smn_applied"
++++
+<![end-metadata]-->
+
+# Dockerizing a Couchbase service
+
+This example shows how to start a [Couchbase](http://couchbase.com) server using Docker Compose, configure it using its [REST API](http://developer.couchbase.com/documentation/server/4.0/rest-api/rest-endpoints-all.html), and query it.
+
+Couchbase is an open source, document-oriented NoSQL database for modern web, mobile, and IoT applications. It is designed for ease of development and Internet-scale performance.
+
+## Start Couchbase server using Docker Compose
+
+Couchbase Docker images are published at: https://hub.docker.com/_/couchbase/.
+
+Use the following Compose file to start Couchbase server:
+
+```
+mycouchbase:
+  image: couchbase/server
+  ports:
+    - 8091:8091
+    - 8092:8092 
+    - 8093:8093 
+    - 11210:11210
+```
+
+Meaning of different ports exposed is at http://developer.couchbase.com/documentation/server/4.1/install/install-ports.html.
```

Comment:
The purpose each exposed port is explained at... 

Also make a link of the URL.